### ISSUE TITLE
Change "Best Audio" from m4a to opus

### ISSUE THREAD

**Issue by gamebeaker** [sylviiu/ezytdl#155]
Hi i wanted to ask if it is possible, that ezytdl downloads if you choose Best Audio the opus version and not the m4a version.
I have no idea about audio codecs so here are a few links to this topic.
[https://stackoverflow.com/questions/49804874/download-the-best-quality-audio-file-with-youtube-dl](https://stackoverflow.com/questions/49804874/download-the-best-quality-audio-file-with-youtube-dl) (not so relevant)
[https://stackoverflow.com/questions/49048859/how-to-download-1080p-best-audio-using-youtube-dl/49067058#49067058](https://stackoverflow.com/questions/49048859/how-to-download-1080p-best-audio-using-youtube-dl/49067058#49067058) (relevant)
[https://www.reddit.com/r/youtubedl/comments/11v8xkk/opus_vs_m4a/](https://www.reddit.com/r/youtubedl/comments/11v8xkk/opus_vs_m4a/) (opus vs m4a)
In summary opus is newer and better but because m4a was more broadly supported from devices it is the "best audio".
These post are a few years old at this point and support for opus is now better.
Is it possible that ezytdl downloads the opus version as "Best Audio" instead of m4a?

Thanks for the project.

**Comment by sylviiu** [comment]
I could totally look into this; I don't know if yt-dlp prioritizes opus over m4a, but if it does, I can look more into using their format sorting algorithm (I only put one of my own in place because the format sorting wouldn't pass through the JSON output)
